Apps/phsmime/smime.json

```diff
@@ -0,0 +1,313 @@
+{
+    "appid": "cd06b954-df08-49cc-aab5-37da164a07c7",
+    "name": "S/MIME",
+    "description": "This app provides support to S/MIME integrations",
+    "type": "email",
+    "product_vendor": "S/Mime",
+    "logo": "logo_smime.svg",
+    "logo_dark": "logo_smime_dark.svg",
+    "product_name": "S/MIME",
+    "product_version_regex": ".*",
+    "publisher": "Splunk Community",
+    "license": "Copyright (c) 2020 Splunk Inc.",
+    "app_version": "1.0.0",
+    "utctime_updated": "2019-12-06T22:45:54.000000Z",
+    "package_name": "phantom_smime",
+    "main_module": "smime_connector.pyc",
+    "min_phantom_version": "4.8.24304",
+    "app_wizard_version": "1.0.0",
+    "pip_dependencies": {
+        "wheel": [
+            {
+                "module": "M2Crypto",
+                "input_file": "wheels/M2Crypto-0.35.2-cp27-cp27mu-linux_x86_64.whl"
+            },
+            {
+                "module": "typing",
+                "input_file": "wheels/typing-3.7.4.1-py2-none-any.whl"
+            }
+        ]
+    },
+    "configuration": {
+        "private_key": {
+            "data_type": "file",
+            "description": "Private Key .pem file",
```

Comment:
nit: can we make the order of `data_type` and `description` consistent

Reply:
Added the suggested change in this commit: https://github.com/phantomcyber/phantom-apps/pull/365/commits/66e66e3db9aba57c4758ff60b82751d9ce57a8b4